CMakeLists.txt

```diff
@@ -519,10 +519,27 @@ add_subdirectory( "cmake-proxies" )
 # Conan uses find_package and does not set GLOBAL flag
 resolve_conan_dependencies()
```

Comment:
Please explain more in the comment of this commit:
* lib-src/portmixer/portaudio.patch described how the customized portaudio used to differ from the upstream.
* Patches were motivated by needs of portmixer, but now portmixer is rewritten all in terms of the published portaudio API.
* libnyquist and portmixer, which use portaudio, also get rewrite of their build system.  (Or could this step be its own separate commit before this commit?  If so, ideally I'd do it, but verifying the rebuilds might be too bothersome now.)
* any other helpful details you think of.

Also amend the commit to delete files in lib-src/portmixer that are no longer needed: the .patch for sure, and maybe also makefiles and other relics of old build systems.  And maybe do the same for libnyquist.